src/systems/inspect-yourself-system.js

```diff
@@ -1,8 +1,9 @@
+import { isLockedDownDemoRoom } from "../utils/hub-utils";
 import { paths } from "./userinput/paths";
 export class InspectYourselfSystem {
   tick(scene, userinput, cameraSystem) {
     if (!scene.is("entered")) return;
-    if (userinput.get(paths.actions.startInspectingSelf)) {
+    if (userinput.get(paths.actions.startInspectingSelf) && !isLockedDownDemoRoom()) {
```

Comment:
Another option would have been to early exit as we do with the `enter` scene condition:
```javascript
if (!scene.is("entered") || isLockedDownDemoRoom()) return;
```